test_settings.py

```diff
@@ -11,17 +11,13 @@
 gettext = lambda s: s
 
 HELPER_SETTINGS = {
-    'NOSE_ARGS': [
-        '-s',
-    ],
-    'ROOT_URLCONF': 'filer.test_utils.urls',
     'INSTALLED_APPS': [
         # filer configuration
         'easy_thumbnails',
         'mptt',
         'filer',
+        'djangocms_text_ckeditor',
         # cmsplugin_filer configuration
-        'cmsplugin_filer_file',
```

Comment:
djangocms-helper adds to installed apps app that was passed in ``run.cms('app_name')`` so to avoid duplicates (i think it doesn't checks if that app is already present in installed apps) I've removed it from installed apps settings.